### PR TITLE
VZ-7963 update metrics tests due to VMO metrics name changes

### PR DESCRIPTION
**Pull request by knelson7**

### CODE REVIEW

File: tests/e2e/metrics/syscomponents/metrics_test.go [hunk 1]
```diff
@@ -31,14 +31,15 @@ const (
...
-	esClusterStatusMetric          = "es_cluster_status"
+
+	vmoFunctionMetric      = "vz_monitoring_operator_reconcile_total"
+	vmoCounterMetric       = "vz_monitoring_operator_deployment_update_total"
+	vmoGaugeMetric         = "vz_monitoring_operator_work_queue_size"
+	vmoTimestampMetric     = "vz_monitoring_operator_configmap_last_successful_timestamp"
+	vaoSuccessCountMetric  = "vz_application_operator_appconfig_successful_reconcile_total"
+	vaoFailCountMetric     = "vz_application_operator_appconfig_error_reconcile_total"
+	vaoDurationCountMetric = "vz_application_operator_appconfig_reconcile_duration_count"
+	esClusterStatusMetric  = "es_cluster_status"
```
Comment (anders-swanson) [inline tests/e2e/metrics/syscomponents/metrics_test.go:42]: Why is it 'es'?

Comment (knelson7) [reply tests/e2e/metrics/syscomponents/metrics_test.go:42]: Ope, yea I guess it should be os? I've been looking for where this metric is being defined and I can't seem to find it. Are there metrics defined outside of metricsexporter/?

Comment (anders-swanson) [reply tests/e2e/metrics/syscomponents/metrics_test.go:42]: This probably is defined in the VMO, this looks like an E2E test that validates the metrics exist across all the verrazzano operators

**Comment by knelson7** [comment]
Closing because Karan already merged a fix https://github.com/verrazzano/verrazzano/pull/5035

**Pull request closed without merging**